### PR TITLE
Gaurav shift size limit

### PR DESCRIPTION
**Pull request by garggaurav**
Created routes, views and controllers for events and shifts. Passing cucumber tests for view_filled_shifts.feature and shift_limit_user.feature.
You can test shifts and events by visiting /shift/:id and /event/:id

### CODE REVIEW

**Comment by garggaurav** [comment]
You can test shifts and events by visiting /shift/:id and /event/:id

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -5,5 +5,68 @@ class ApplicationController < ActionController::Base
 
   def index
     @user = User.find_by id: session[:user_id]
+
+    #Seeding db for testing purpose. DELETE IN PRODUCTION
```
Comment (virajmahesh) [inline app/controllers/application_controller.rb]: This shouldn't be here. There is no way we can delete this just in production. If you need to create objects for tests, do it inside the test code.

File: spring-1000/7db707a8a02603bc63468a82a7013771.pid [hunk 2]
```diff
@@ -0,0 +1 @@
+3781
```
Comment (virajmahesh) [inline spring-1000/7db707a8a02603bc63468a82a7013771.pid]: Remove this file.

File: features/step_definitions/custom_steps.rb [hunk 3]
```diff
@@ -26,4 +26,56 @@
...
+  Event.create(:id => 1, :user_id => 1, :description => "Go Bernie!", :location => "Berkeley", :candidate => "Bernie Sanders")
+end
+
+Given(/^event (\d+) has (\d+) shifts with (\d+) spaces available$/) do |eventID, numShifts, limit|
+  (1..numShifts.to_i).each do |index|
+    Shift.create(:start_time => Time.now, :end_time => Time.now, :event_id => eventID,:limit => limit.to_i,:has_limit => true, :role => 'Tabler')
+  end
+end
+
+Given(/^(.*) visit the event (\d+) page$/) do |arg1, eventID|
```
Comment (virajmahesh) [inline features/step_definitions/custom_steps.rb]: nit: If you don't need arg1 then can you get rid of it?

**Pull request merged**